<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="canvas_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #eee;
border: 1px solid #444;
</string>
      </property>
      <property name="text">
       <string>CANVAS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="undoButton">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="redoButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Redo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="designSelectBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Custom Design</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MacEng Fireball</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>McMaster Logo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Smiley Face</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #0d6efd;
color: #fff;</string>
        </property>
        <property name="text">
         <string>Send to Dotter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #dc3545;
color: #fff;
</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_loadDesign"/>
    <addaction name="action_saveDesign"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionLoad_Canvas_Template"/>
    <addaction name="actionClear_Canvas_Template"/>
    <addaction name="separator"/>
    <addaction name="action_homeMachine"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_saveDesign">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="action_loadDesign">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="action_connectDotter">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="action_disconnectDotter">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="action_dotterStatus">
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="action_dotterSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>Custom...</string>
   </property>
  </action>
  <action name="actionRainbow">
   <property name="text">
    <string>Rainbow</string>
   </property>
  </action>
  <action name="action_penColorPerLayer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pen Color Per Layer</string>
   </property>
  </action>
  <action name="actionClear_Canvas">
   <property name="text">
    <string>Clear Canvas</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="action_clearCanvas">
   <property name="text">
    <string>Clear Canvas</string>
   </property>
  </action>
  <action name="action_penColorPerLayer_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pen Color Per Layer</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionLoad_Canvas_Template">
   <property name="text">
    <string>Load Canvas Template...</string>
   </property>
  </action>
  <action name="actionClear_Canvas_Template">
   <property name="text">
    <string>Clear Canvas Template</string>
   </property>
  </action>
  <action name="action_homeMachine">
   <property name="text">
    <string>Home Machine</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
